app/va/vetext/client.py

```diff
@@ -105,12 +116,21 @@ def _decode_bad_request_response(
         self,
         http_exception,
     ):
+        """Parse the response and raise an exception as this is always an exception
+
+        Args:
+            http_exception (Exception): The exception raised
+
+        Raises:
+            NonRetryableException: Raised exception
+        """
         try:
             payload = http_exception.response.json()
-        except Exception:
-            message = http_exception.response.text
-            raise VETextBadRequestException(message=message) from http_exception
-        else:
             field = payload.get('idType')
             message = payload.get('error')
-            raise VETextBadRequestException(field=field, message=message) from http_exception
+            self.logger.warning('Bad response from VEText: %s with field: ', message, field)
+            raise NonRetryableException from http_exception
+        except Exception:
+            message = http_exception.response.text
+            self.logger.warning('Bad response from VEText: %s', message)
+            raise NonRetryableException from http_exception
```

Comment:
should these be error logs instead of warning logs?

Reply:
Yes - I think that makes more sense. Updated.